.github/workflows/build-and-publish.yaml

```diff
@@ -0,0 +1,45 @@
+name: build-and-publish
+on: [push]
+jobs:
+  build-and-publish:
+    name: Build and Publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
```

Comment:
could we please continue to use podman instead?

Reply:
+1 on podman usage.

Reply:
Need to use docker for QEMU action as docker can successfully run multi-arch builds at once without the use of using manifests